lib/order.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :basket, :dishes
+
+  def initialize
+    @basket = []
+    @dishes = {
+      "Margherita" => 5.00,
+      "Pepperoni" => 8.00
+    }
```

Comment:
In order initialize, you are creating the menu again.  All you need to do here is create @dishes as an instance of menu and you could access the menu through the attribute reader.

Reply:
Thanks, have amended this to '@dishes = Menu.new' so that it calls the menu from the Menu class file.